src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -309,15 +310,15 @@ public void testIsSpecialFile() {
     }
 
     @Test( dataProvider = "ByteArrayIOTestData" )
-    public void testWriteThenReadFileIntoByteArray ( int fileSize, int readBufferSize ) throws Exception {
+    public void testWriteThenReadFileIntoByteArray ( int fileSize, int readBufferSize ) {
         File tempFile = createTempFile(String.format("testWriteThenReadFileIntoByteArray_%d_%d", fileSize, readBufferSize), "tmp");
 
         byte[] dataWritten = getDeterministicRandomData(fileSize);
         IOUtils.writeByteArrayToFile(dataWritten, tempFile);
         byte[] dataRead = IOUtils.readFileIntoByteArray(tempFile, readBufferSize);
 
         Assert.assertEquals(dataRead.length, dataWritten.length);
-        Assert.assertTrue(Arrays.equals(dataRead, dataWritten));
+        Assert.assertEquals(dataWritten, dataRead);
```

Comment:
Order of args seems wrong here -- `dataRead` is the "actual" value here, isn't it?

Reply:
yes, good catch